force-app/main/default/pages/STG_Settings.page

```diff
@@ -3,45 +3,39 @@
     <script src="/soap/ajax/48.0/apex.js" type="text/javascript"></script>
     <script src="/lightning/lightning.out.js" type="text/javascript"></script>
 
-    <div class="slds">
+   <div class="slds">
         <!-- layout=block makes the outputPanel produce a DIV -->
-        <apex:outputPanel layout="block" id="stgContainer" rendered="{!hasPerm}"></apex:outputPanel>
-        
-        <!-- No Access without Profile.PermissionsCustomizeApplication -->
-        <apex:outputPanel layout="block" rendered="{!NOT(hasPerm)}">
-            <h1 style="font-size: 1.5em;">You do not have permission to view this page.</h1> 
-            <p>Please contact your administrator if you require access.</p>
-        </apex:outputPanel>
-    </div>
+        <apex:outputPanel layout="block" id="stgContainer"></apex:outputPanel>
+    </div> 
 
     <script>
         //Because sometimes the session ID is not set correctly
         sforce.connection.sessionId = "{!$Api.Session_ID}";
 
         // We dont know the ID (because visualforce) so were gonna use a CSS selector to get it
         var stgContainerId = document.querySelector('[id$="stgContainer"]').id;
-        
+
         var namespace = '';
         var namespacePrefix = '';
         //This should be a class unique to EDA so the namespace comes through correctly
-        var gettingnamespace = sforce.connection.query("SELECT NamespacePrefix FROM ApexClass where Name = 'CCON_Faculty_TDTM' LIMIT 1"); 
+        var gettingnamespace = sforce.connection.query("SELECT NamespacePrefix FROM ApexClass where Name = 'STG_Base_CTRL' LIMIT 1"); 
```

Comment:
@MatthewBlanski is it ok if we change it to 'STG_Base_CTRL'? we were wondering it this would be unique or not if there are multiple packages... cc: @xinnanlisf

Reply:
Yes